code/style.css

```diff
@@ -1,7 +1,233 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  background-color: rgba(58, 116, 114, 0.24);
+}
+
+/*NAVBAR*/
+ul {
```

Comment:
Remember - we shouldn't style by anything else than CSS classes. Let's add classes in HTML and style them over here